ansible/roles/zabbix_es/templates/ESzabbix.py.j2

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
+
+# Created by Aaron Mildenstein on 19 SEP 2012
+# Switchted from pyes to Elasticsearch for better Health Monitoring by Marcel Alburg on 17 JUN 2014
+# Adjusted for ELK 2.3 by Javier Barroso on 17 AUG 2016
+
+from elasticsearch import Elasticsearch
+
+import sys
+import json
+import shelve
+import os
+import time
+
+# Define the fail message
+def zbx_fail():
+    print "ZBX_NOTSUPPORTED"
+    sys.exit(2)
+
+def use_cache(file):
+    if not os.access(file,os.F_OK):
+        return False
+    now = int(time.time())
+    file_modified = int(os.stat(file).st_mtime)
+    if now - file_modified < 60:
+       return True
+    else:
+       return False
+    
+    
+searchkeys = ['query_total', 'fetch_time_in_millis', 'fetch_total', 'fetch_time', 'query_current', 'fetch_current', 'query_time_in_millis']
+getkeys = ['missing_total', 'exists_total', 'current', 'time_in_millis', 'missing_time_in_millis', 'exists_time_in_millis', 'total']
+docskeys = ['count', 'deleted']
+indexingkeys = ['delete_time_in_millis', 'index_total', 'index_current', 'delete_total', 'index_time_in_millis', 'delete_current']
+storekeys = ['size_in_bytes', 'throttle_time_in_millis']
+cachekeys = ['filter_size_in_bytes', 'field_size_in_bytes', 'field_evictions']
+clusterkeys_direct = docskeys + storekeys
+clusterkeys_indirect = searchkeys + getkeys + indexingkeys 
+returnval = None
+conn = None
+user=str(os.getuid())
+clustercache_file = "/tmp/clusterstats.cache." + user
+nodescache_file = "/tmp/nodestats.cache." + user
+lock_file="/tmp/ESzabbix.lock." + user
+
+# now = time.time()
+# logging = open("/tmp/whathappens.log."+str(now)+"-"+sys.argv[1]+"-"+sys.argv[2], "w");
```

Comment:
seems like this logging mechanism requires you to uncomment all the log statements. feel like it'd be better to have a `--debug` flag instead